upgrade_test.py

```diff
@@ -439,231 +439,233 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
-
-        # generate random order to upgrade
-        nodes_num = len(self.db_cluster.nodes)
-        # prepare an array containing the indexes
-        indexes = list(range(nodes_num))
-        # shuffle it so we will upgrade the nodes in a random order
-        random.shuffle(indexes)
-
-        self.log.info('pre-test - Run stress workload before upgrade')
-        # complex workload: prepare write
-        self.log.info('Starting c-s complex workload (5M) to prepare data')
-        stress_cmd_complex_prepare = self.params.get('stress_cmd_complex_prepare')
-        complex_cs_thread_pool = self.run_stress_thread(
-            stress_cmd=stress_cmd_complex_prepare, profile='data_dir/complex_schema.yaml')
-
-        # wait for the complex workload to finish
-        self.verify_stress_thread(complex_cs_thread_pool)
-
-        # prepare write workload
-        self.log.info('Starting c-s prepare write workload (n=10000000)')
-        prepare_write_stress = self.params.get('prepare_write_stress')
-        prepare_write_cs_thread_pool = self.run_stress_thread(stress_cmd=prepare_write_stress)
-        self.log.info('Sleeping for 60s to let cassandra-stress start before the upgrade...')
-        time.sleep(60)
-
-        with ignore_upgrade_schema_errors():
-
-            step = 'Step1 - Upgrade First Node '
-            self.log.info(step)
-            # upgrade first node
-            self.db_cluster.node_to_upgrade = self.db_cluster.nodes[indexes[0]]
-            self.log.info('Upgrade Node %s begin', self.db_cluster.node_to_upgrade.name)
-            self.upgrade_node(self.db_cluster.node_to_upgrade)
-            self.log.info('Upgrade Node %s ended', self.db_cluster.node_to_upgrade.name)
-            self.db_cluster.node_to_upgrade.check_node_health()
-
-            # wait for the prepare write workload to finish
-            self.verify_stress_thread(prepare_write_cs_thread_pool)
-
-            # read workload (cl=QUORUM)
-            self.log.info('Starting c-s read workload (cl=QUORUM n=10000000)')
-            stress_cmd_read_cl_quorum = self.params.get('stress_cmd_read_cl_quorum')
-            read_stress_queue = self.run_stress_thread(stress_cmd=stress_cmd_read_cl_quorum)
-            # wait for the read workload to finish
-            self.verify_stress_thread(read_stress_queue)
-            self.fill_and_verify_db_data('after upgraded one node')
-            self.search_for_idx_token_error_after_upgrade(node=self.db_cluster.node_to_upgrade,
-                                                          step=step+' - after upgraded one node')
 
-            # read workload
-            self.log.info('Starting c-s read workload for 10m')
-            stress_cmd_read_10m = self.params.get('stress_cmd_read_10m')
-            read_10m_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_read_10m)
+        with EventsFilter(event_class=PrometheusAlertManagerEvent, regex="CassandraStressWriteTooSlow"):
```

Comment:
Why do you want to ignore the event?
Is it flooded in the sct log or event log?

@fruch Can we suppress this kind of message/event and output a summary in a period?

like: raise CassandraStressWriteTooSlow event 100 times in one minute

Reply:
It’s not flooding, it sometimes fails the test.
I Don’t think we should ignore this alert, and even if we do, we shouldn’t use it with filter, we should configure the alert manager with higher value.